var/spack/repos/builtin/packages/py-pybtex-docutils/package.py

```diff
@@ -12,6 +12,9 @@ class PyPybtexDocutils(PythonPackage):
     homepage = "https://pypi.python.org/pypi/pybtex-docutils/"
     url      = "https://pypi.io/packages/source/p/pybtex-docutils/pybtex-docutils-0.2.1.tar.gz"
 
+    import_modules = ['pybtex_docutils']
```

Comment:
Spack now automatically detects `import_modules`, this might not be needed anymore